cmd/goprotoc/codegen.go

```diff
@@ -166,14 +183,18 @@ func driveProtocAsPlugin(req *plugins.CodeGenRequest, resp *plugins.CodeGenRespo
 		return err
 	}
 
-	args := make([]string, 2+len(req.Files)+len(req.Args))
-	args[0] = "--descriptor_set_in=" + descFile
-	args[1] = "--" + lang + "_out=" + outDir
-	for i, arg := range req.Args {
-		args[i+2] = arg
+	args := make([]string, 0, 2+len(req.Files)+len(req.Args))
+	args = append(args, "--descriptor_set_in="+descFile)
+	args = append(args, "--"+lang+"_out="+outDir)
+	for _, arg := range req.Args {
+		if arg != "" {
```

Comment:
Should these instead return errors instead of silently skipping them?

Reply:
Either/or - I think a good argument could be made for erroring. I'll change to that.

This happened from Prototool, which means I have a bug in Prototool.